algorithms/schema-free/kmeans_sub.gsql

```diff
@@ -0,0 +1,129 @@
+CREATE QUERY kMeans_sub(int k, float maxChange = 1.0, vertex input, string e_type, bool random = FALSE, BOOL print_accum = TRUE, STRING result_attr="", STRING                    file_path="") returns (ListAccum<Float>) {
+  TypeDef Tuple<int center, float value> distanceTuple;
+  HeapAccum<distanceTuple> (1,value asc) @distanceHeap;
+  ArrayAccum<AvgAccum> @@Array[100][100];
```

Comment:
Make the index numbers generic. taking the size from the input parameter

Reply:
not currently possible